airflow/docker.go

```diff
@@ -703,13 +707,13 @@ func (d *DockerCompose) versionTest(testHomeDirectory, currentAirflowVersion, de
 	return nil
 }
 
-func (d *DockerCompose) dagTest(testHomeDirectory, newAirflowVersion, newDockerFile, customImage, buildSecretString string) error {
+func (d *DockerCompose) dagTest(testHomeDirectory, newAirflowVersion, newDockerFile, customImage, buildSecretString string) (string, error) {
```

Comment:
Can we return an int instead of a string?

Reply:
The exit code is parsed from the docker logs so it ends up as a string